autoscaler/cluster.py

```diff
@@ -639,3 +648,7 @@ def maintain(self, cached_managed_nodes, running_insts_map,
                         else:
                             logger.info(
                                 '[Dry run] Would have terminated unmanaged %s', inst)
+
+    def _map_sets_to_values(self, set_map):
+        for key in set_map.keys():
+            set_map[key] = int(min(set_map[key]))
```

Comment:
I was going to say that it's fine for behavior in the case to be undefined or that it would be fine to just find the min. But the more I think about it, I'm leaning towards throwing an error. We should probably warn the user they're trying to give the same instance type two different priorities..